tests/test_data/ru/test_russia_data.py

```diff
@@ -0,0 +1,33 @@
+import re
+import typing
+
+import pytest
+
+from mimesis import Person
+from mimesis.locales import Locale
+
+NON_RU_LETTERS = re.compile(r"(?:(?![а-яё\d])[\w])+", re.IGNORECASE)
+
+
+def params(data: typing.Dict):
+    def extract_data(data: typing.Dict, result: typing.List, key: str = '') -> None:
```

Comment:
Please, remove type annotations. We don't annotate tests.

Reply:
Fixed